makefile

```diff
@@ -166,7 +169,6 @@ endif
 # Restyles all C++ (Clang formatter) and Python files (Black formatter) excluding files in the external and build folders
```

Comment:
is there a way to only run black on python files? I had issues where black and clang-format disagreed, resulting in an inability to commit

otherwise this is an okay solution because out pre-commit hooks force us to conform to both black and pretty-lines separately (which works when they don't conflict)

```suggestion
# Restyles all C++ (Clang formatter) excluding files in the external and build folders. For Python, run black rj_gameplay.
```